Makefile

```diff
@@ -3,20 +3,22 @@
 #
 all: binary
 
+PKGS := $(shell go list ./... | grep -vE '/vendor/|github.com/docker/cli/man|github.com/docker/cli/docs/yaml$$')
```

Comment:
@dnephin sure, I will simply duplicate it in this Makefile, but I do want to remove `grep -v vendor` in unit-test-with-coverage, and replace it with `$@`.